ToDoGeo/ToDoGeo/Presenter/Flows/SignInFlow.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  SignInFlow.swift
+//  ToDoGeo
+//
+//  Created by SUN on 4/21/24.
+//
+
+import UIKit
+import RxFlow
+
+final class SignInFlow: Flow {
+    var root: Presentable {
+        return rootViewController
+    }
+    
+    private lazy var rootViewController: UINavigationController = {
+        let viewController = UINavigationController()
+        return viewController
```

Comment:
이 부분은 아래와 같이 한 줄로 처리했어도 되었을 것 같은데요. 어떻게 생각하시나요? 

```suggestion
        return UINavigationController()
```

Reply:
[198f0f7] 수정한 커밋입니다.